pkg/controllers/provisioning/suite_test.go

```diff
@@ -246,6 +247,31 @@ var _ = Describe("Provisioning", func() {
 			ExpectScheduled(ctx, env.Client, pod)
 		}
 	})
+	It("should not use a different NodePool hash on the NodeClaim if the NodePool changes during scheduling", func() {
```

Comment:
I'm not following - where is the nodepool changed in the test?

Reply:
lol good point. I'm an amazing test writer :D